.changeset/few-dryers-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+"haunted": minor
+---
+
+New hook: useProvideContext
```

Comment:
this should be documented in readme and on the docs site

Reply:
@bennypowers I have added documentation for the hook.
Do I have to also update custom-elements.json (with the results of `npm run analyze`)?